### DISCUSSION TITLE
E-Jenda - Blog - GrahamSH

### DISCUSSION THREAD

**Discussion by giscus** [GrahamSH-LLK/grahamsh.com#18] [General]
# E-Jenda - Blog - GrahamSH

Introducing my newest project: the E-Jenda

https://grahamsh.com/blog/e-jenda/

**Comment by GrahamSH-LLK** [comment]
Update: Check it out at https://ejenda.grahamsh.com

**Reply by Cool-Scratcher** [reply]
epic

**Reply by VeryUsual** [reply]
Cool

**Reply by GrahamSH-LLK** [reply]
Update update: Check it out at https://ejenda.org
